docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,631 @@ license: |
+    </tr>
+    <tr>
+      <td><b>covar_pop</b>(<i>expression1, expression2</i>)</td>
+      <td>(double, double)</td>
+      <td>Returns the population covariance of a set of number pairs.</td>
+    </tr> 
+    <tr>
+      <td><b>covar_samp</b>(<i>expression1, expression2</i>)</td>
+      <td>(double, double)</td>
+      <td>Returns the sample covariance of a set of number pairs.</td>
+    </tr>  
+    <tr>
+      <td><b>{first | first_value}</b>(<i>expression[, `isIgnoreNull`]</i>)</td>
+      <td>(any[, boolean])</td>
+      <td>Returns the first value of expression for a group of rows. If isIgnoreNull is true, returns only non-null values, default is false. This function is non-deterministic.</td>
+    </tr>      
+    <tr>
+      <td><b>kurtosis</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the kurtosis value calculated from values of a group.</td>
+    </tr>    
+    <tr>
+      <td><b>{last | last_value}</b>(<i>expression[, `isIgnoreNull`]</i>)</td>
+      <td>(any[, boolean])</td>
+      <td>Returns the last value of expression for a group of rows. If isIgnoreNull is true, returns only non-null values, default is false. This function is non-deterministic.</td>
+    </tr>      
+    <tr>
+      <td><b>max</b>(<i>expression</i>)</td>
+      <td>tinyint|short|int|bigint|float|double|date|timestamp|string, or arrays of these types</td>
+      <td>Returns the maximum value of the expression.</td>
+    </tr>          
+    <tr>
+      <td><b>max_by</b>(<i>expression1, expression2</i>)</td>
+      <td>tinyint|short|int|bigint|float|double|date|timestamp|string, or arrays of these types</td>
+      <td>Returns the value of expression1 associated with the maximum value of expression2.</td>
+    </tr>   
+    <tr>
+      <td><b>min</b>(<i>expression</i>)</td>
+      <td>tinyint|short|int|bigint|float|double|date|timestamp|string, or arrays of these types</td>
+      <td>Returns the minimum value of the expression.</td>
+    </tr>          
+    <tr>
+      <td><b>min_by</b>(<i>expression1, expression2</i>)</td>
+      <td>tinyint|short|int|bigint|float|double|date|timestamp|string, or arrays of these types</td>
+      <td>Returns the value of expression1 associated with the minimum value of expression2.</td>
+    </tr>      
+    <tr>
+      <td><b>percentile</b>(<i>expression, percentage [, frequency]</i>)</td>
+      <td>(short|float|byte|decimal|double|int|bigint, double[, int])</td>
+      <td>`percentage` is a number between 0 and 1; `frequency` is a positive integer. Returns the exact percentile value of numeric expression at the given percentage.</td>
+    </tr>         
+    <tr>
+      <td><b>percentile</b>(<i>expression, <b>array</b>(percentage1 [, percentage2]...) [, frequency]</i>)</td>
```

Comment:
I think we don't need to say `array` here because you said `array of double` in the argument section below.